[     1.490][MainProcess][INFO] Parsing specification...
[     1.493][MainProcess][INFO] File read
[     1.494][MainProcess][DEBUG] Reading input files...
[     1.501][MainProcess][INFO] Inferred data types for table tests-examples/textbook/tables/aircraft.txt: ['int64', 'string', 'int64']
[     1.511][MainProcess][INFO] Inferred data types for table tests-examples/textbook/tables/flights.txt: ['int64', 'string', 'string', 'int64', 'datetime64[ns]', 'datetime64[ns]', 'int64']
[     1.515][MainProcess][INFO] Inferred data types for table tests-examples/textbook/tables/certified.txt: ['int64', 'int64']
[     1.520][MainProcess][INFO] Inferred data types for table tests-examples/textbook/tables/employees.txt: ['int64', 'string', 'Float64']
Traceback (most recent call last):
  File "sequential.py", line 84, in <module>
    main()
  File "sequential.py", line 46, in main
    specification = Specification(spec)
  File "/home/vbernardo/SKEL/squares/dsl/specification.py", line 94, in __init__
    self.data_frames['expected_output'] = self.read_table(self.output, 'expected_output')
  File "/home/vbernardo/SKEL/squares/dsl/specification.py", line 131, in read_table
    df = pandas.read_csv(path)
  File "/home/vbernardo/miniconda3/envs/SKEL/lib/python3.8/site-packages/pandas/io/parsers.py", line 610, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/vbernardo/miniconda3/envs/SKEL/lib/python3.8/site-packages/pandas/io/parsers.py", line 462, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/vbernardo/miniconda3/envs/SKEL/lib/python3.8/site-packages/pandas/io/parsers.py", line 819, in __init__
    self._engine = self._make_engine(self.engine)
  File "/home/vbernardo/miniconda3/envs/SKEL/lib/python3.8/site-packages/pandas/io/parsers.py", line 1050, in _make_engine
    return mapping[engine](self.f, **self.options)  # type: ignore[call-arg]
  File "/home/vbernardo/miniconda3/envs/SKEL/lib/python3.8/site-packages/pandas/io/parsers.py", line 1867, in __init__
    self._open_handles(src, kwds)
  File "/home/vbernardo/miniconda3/envs/SKEL/lib/python3.8/site-packages/pandas/io/parsers.py", line 1362, in _open_handles
    self.handles = get_handle(
  File "/home/vbernardo/miniconda3/envs/SKEL/lib/python3.8/site-packages/pandas/io/common.py", line 642, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: 'tests-examples/textbook/tables/32.out'
